src/main/java/org/sonarsource/sonarlint/ls/SonarLintLanguageServer.java

```diff
@@ -678,6 +678,10 @@ public Map<String, Path> getEmbeddedPluginsToPath() {
     addPluginPathOrWarn("xml", Language.XML, plugins);
     addPluginPathOrWarn("text", Language.SECRETS, plugins);
     addPluginPathOrWarn("go", Language.GO, plugins);
+    addPluginPathOrWarn("iac", Language.CLOUDFORMATION, plugins);
+    addPluginPathOrWarn("iac", Language.DOCKER, plugins);
+    addPluginPathOrWarn("iac", Language.KUBERNETES, plugins);
+    addPluginPathOrWarn("iac", Language.TERRAFORM, plugins);
```

Comment:
As far as I can tell, only one call to `addPluginPathOrWarn` is needed (see e.g how CFamily is added above).